package.json

```diff
@@ -121,6 +119,8 @@
     "typescript": "^3.0.3"
   },
   "dependencies": {
+    "@daostack/migration": "0.0.1-rc.7-v2",
```

Comment:
how about using 0.0.1-rc.8-v3 ?

Reply:
sure but not in this release :-)

Reply:
https://github.com/daostack/client/issues/71